OlxProject/pages/user_page.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../tools/all_requires'
+
+class UserPage
+  include TopPart
+  USER_PAGE_MESSAGE = "Объявления"
+
+  protected def observed_search_link
+    $driver.find_element(id: 'observed-search-link')
+  end
+
+  def observed_search_link_click
+    observed_search_link().click
+  end
+
+  def get_observed_item
+    $driver.find_element(xpath: "//span[contains(@class,'block fbold')]").text
+  end
```

Comment:
get_observed_item method name intuitively tells that it will return an item, not it's text. Maybe additional method for text is needed?

Reply:
I need only method which return .text, so I renamed this method.